armi/reactor/tests/test_blocks.py

```diff
@@ -307,6 +306,8 @@ def getComponentData(component):
 
 class TestDetailedNDensUpdate(unittest.TestCase):
     def setUp(self):
+        from armi.reactor.blueprints.tests.test_blockBlueprints import FULL_BP
```

Comment:
Why move this down here?

Reply:
While testing this PR I found an infinite import loop in ARMI.

It's between `test_blocks.py` and `test_blockBlueprints.py`.

It turns out Python usually figures it's way around it, but I hit a situation (running just certain tests in parallel), where it didn't.

So I fixed the infinite import loop.  Unrelated to crumbs, obviously.

Reply:
Huh. Cool! Good stuff. 

Does it make sense to move it to a `setUpClass` class method so the import isn't done for every single test?

Reply:
I mean, that entire class only has ONE test.  So, it's not a big deal.

And that test class isn't imported or subclasses anywhere.

Reply:
In fact, I am just going to remove the `setUp()` method from that class. It's just one test that needs to be independent from others anyway.